app/views/observer_mailer/observer_added_notification.text.erb

```diff
@@ -1,6 +1,6 @@
 <%= observer_text(@observation, @reason) %>
 
-<%= t("mailer.proposal_link_text",
-      proposal_url: proposal_url(@proposal)) %>
+<%= t("mailer.view_request_cta") %>
+<%= proposal_url(@proposal) %>
```

Comment:
Seeing this change again. Is this intentional or is this because it hasnt been reviewed yet?